integration/tools/validation/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -228,6 +235,9 @@ private int loadAndRunTests(Object operations, String testDir, PrintStream msgSt
       for (Method test : tests) {
         String testName = test.getName();
         if (testName.endsWith("Test")) {
+          if (!(mTestList.isEmpty() || mTestList.contains(testName))) {
```

Comment:
will be better for us to check if the input test names are valid. if not, print out all valid test names. it doesn't need to be done if current PR

Reply:
FYI, if you don't have time to do this, can create a contributor task for other people to work on it

Reply:
https://github.com/Alluxio/new-contributor-tasks

@LuQQiu Good suggestion, created a new contributor task.